tests/storages_tests/rdb_tests/create_db_v3.py

```diff
@@ -0,0 +1,66 @@
+from argparse import ArgumentParser
```

Comment:
This is not a strong opinion, as those are utility scripts or I might have missed something, but I'm wondering if we could just add flag to `create_db.py` which would enable additional params in objective functions for `2.6.0.db` up. This would also reduce code in tests, since we could use existing parametrization.

Reply:
Thank you for the insightful suggestion. I'll try to make the script unify!

Reply:
Unified in 3b314c1. :eyes: